fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckInfoCard.kt

```diff
@@ -137,6 +137,7 @@ enum class ReviewQualityCheckInfoType {
     Confirmation,
     Error,
     Info,
+    Analysis,
```

Comment:
where in the figma is this card defined?

Is this possibly from a retired spec? I noticed the figma in the root comment of the ticket leads to a "graveyard" page. Here is a [link to the up-to-date specs](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?type=design&node-id=2-4&mode=design&t=3bRXCHiytlvqO5k6-0) and here's a [link to the component this composable is derived from](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?type=design&node-id=5384-23074&mode=design&t=3bRXCHiytlvqO5k6-0).

Reply:
So I found the answer to my own question after going through Verdi's video from Friday. To close that loop, this is from the [updated re-analysis flow specs](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?type=design&node-id=7427-62855&mode=design&t=PJIXBFg0yjW0tmHr-0).

Reply:
For this flow, let's do the following:
- (Maybe) rename `Analysis` to `AnalysisUpdate` (don't feel strongly on this, it _might_ make it more clear what this card type is for.
- Make the background color transparent. Compose doesn't have the same issue of color re-drawing that XML does, but making it the explicit `Color.Transparent` makes it more clear (pun intended) that there shouldn't be a background color for this info card type.
- Add a line to the `buttonTextColor` `when` and make it `FirefoxTheme.colors.textActionSecondary` as it is in the spec.

Reply:
I definitely should've included the updated links to the specs, next time. 
Updated AnalysisUpdate, background color, and button text color